main-app/app/index.html

```diff
@@ -1,14 +1,53 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" ng-app="Tombola.NoughtAndCrosses">
 <head>
     <meta charset="UTF-8">
     <title></title>
+    <!--<script src="../../bower_components/angular/angular.min.js"></script>-->
+    <!--<script src="scripts/temp.js"></script>-->
+    <script src="../../thirdparty/bower_components/angular/angular.min.js"></script>
+    <script src="scripts/app.js"></script>
+    <link rel="stylesheet" href="css/main.css">
 </head>
-<body>
-    <img src="images/Michael-Komarck-Knife-of-Dreams.jpg" alt="wheel of time image">
-    <audio controls loop>
-        <source src="sounds/Silly_Farts-Joe-1473367952.mp3" type="audio/mp3">
-        <h1>hello</h1>
-    </audio>
+<body ng-controller="GameboardController">
+    <section>
+        <div ng-click="selectOptionsForPlayer1()" class="{{player1}}">
+            <!--<h1>Player1 : {{player1}}</h1>-->
+        </div>
+        <div ng-click="selectOptionsForPlayer2()" class="{{player2}}">
+            <!--<h1>Player2 : {{player2}}</h1>-->
```

Comment:
Do not leave commented HTML in PRs: the master branch needs to be theoretically deployable at all times